ProcessMaker/Http/Controllers/ProfileController.php

```diff
@@ -6,10 +6,13 @@
 use ProcessMaker\Models\User;
 use ProcessMaker\Models\JsonData;
 use ProcessMaker\i18nHelper;
+use ProcessMaker\Traits\HasControllerAddons;
```

Comment:
Does it make sense to add this here? Maybe in the parent controller would be a better fit?

Reply:
In principle I was trying to limit the scope of addons. I moved this code to the layout and base controller so that any page can have addons now.